kobo/client/__init__.py

```diff
@@ -93,13 +93,20 @@ def run(self, *args, **kwargs):
 )
 
 
-class ClientCommandContainer(kobo.cli.CommandContainer):
-
-    def __init__(self, conf, **kwargs):
+class BaseClientCommandContainer(kobo.cli.CommandContainer):
```

Comment:
Could we get a comment or doc string summarizing the difference between these two classes?  As it is, it's not clear what functionality should go into Base and what doesn't.

For example it's not clear why set_hub isn't in Base.